omod/src/main/webapp/resources/scripts/jquery/autocomplete/OpenmrsAutoComplete.js

```diff
@@ -231,30 +231,40 @@ function CreateCallback(options) {
 		// between the image and the identifier
 		var textShown = " ";
 		
-		if (person.identifier)
-			textShown += person.identifier;
-		
+		if (person.identifier) {
+			textShown += $j('<div/>').text(person.identifier).html();
+		}
+
 		textShown += " ";
 		
-		textShown += person.personName;
+		var personName = "";
+		if (person.givenName !== "null") {
```

Comment:
Why put null in double quotes?

Reply:
The Person object currently stores these values as strings
```
{
    "address1": null,
    "address2": null,
    "age": 0,
    "attributes": {},
    "birthdate": "2024-07-01T18:30:00.000Z",
    "birthdateEstimated": false,
    "birthdateString": "Jul 2, 2024",
    "deathDateString": null,
    "deathdateEstimated": false,
    "familyName": "Doe",
    "gender": "M",
    "givenName": "John",
    "identifier": "1aaa",
    "identifierTypeName": "",
    "middleName": "null",
    "otherIdentifiers": "",
    "otherNames": "",
    "patientId": 4,
    "personId": 4,
    "personName": "John null Doe",
    "uuid": "152075b7-b5b0-4b6f-b9de-0d07b4099afb",
    "voided": false
}
```